scripts/release-manager/package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "chalk": "^1.1.3",
     "colors": "^1.1.2",
+    "es6-promisify": "^4.1.0",
```

Comment:
We can revert this and just remove the dead require I added. I bet that's why lint is unhappy too…